author.rb

```diff
@@ -0,0 +1,16 @@
+class Author
+  attr_accessor :first_name, :last_name
+
+  def initialize(id, first_name, last_name, items = [])
+    @first_name = first_name
+    @last_name = last_name
+    private
+    @items = items
+    @id = id
+  end
+
+  def add_item(item)
+    @items.push(item)
+    item.author = self
+  end
+end
```

Comment:
the id parameter should be autogenerated during the initial instantiation of the author class, u can either use random function or use UUID to get a unique id for each instance of the class